### PR TITLE
optimize docs add lite scaffold

### PR DESCRIPTION
**Pull request by imsobear**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,29 @@
-# 自定义物料初始模板
+# CRA Materials
 
-## 简介
+Materials for Create React App. Make CRA use in iceworks.
```
Comment (tanhengyeow) [inline README.md]: Materials for Create React App. ~~Make CRA use in iceworks.~~

File: README.md [hunk 2]
```diff
@@ -1,37 +1,29 @@
...
-$ npm install
-$ npm run start
+Start iceworks:
+
+```bash
+$ iceworks  # will open http://127.0.0.1:8000 in browser
 ```
 
-#### 模板预览
+Add CRA materials, go to Materials Tab and click Add Button:
```
Comment (tanhengyeow) [inline README.md]: To add CRA materials, go to the `Materials` tab > click `Add`.

File: README.md [hunk 3]
```diff
@@ -1,37 +1,29 @@
...
 
-```
-$ cd scaffolds/ExampleScaffold
-$ npm install
-$ npm run start
-```
+![](https://img.alicdn.com/tfs/TB1o5j2bYr1gK0jSZR0XXbP8XXa-2872-1634.jpg)
 
-## 添加物料
+Fill out the form. CRA materials url `http://ice.alicdn.com/assets/materials/cra-materials.json`
```
Comment (tanhengyeow) [inline README.md]: Fill ~~out~~ up the form. CRA materials **source** url `http://ice.alicdn.com/assets/materials/cra-materials.json`

**Review by tanhengyeow — APPROVED** [review]
Looks good, just some nits :)

**Comment by imsobear** [comment]
@tanhengyeow Fixed 🥇

**Pull request merged**